Makefile

```diff
@@ -105,29 +105,29 @@ codespell: ##             - Runs codespell to check for spelling errors
 	@codespell -S tmp,.git -L te -w
 
 clean: ##             - Removes generated files and dirs
-	rm -f *~  *.tc *.pp *.pp.bz2
-	rm -rf tmp *.tar.gz ${RPM_TOPDIR}
+	@rm -f *~  *.tc *.pp *.pp.bz2
+	@rm -rf tmp *.tar.gz ${RPM_TOPDIR}
 
 man: qm.8.md ##             - Generates the QM man page
 	go-md2man --in qm.8.md --out qm.8
 
 .PHONY: dist
 dist: ##             - Creates the QM distribution package
 	tar cvz \
-		--exclude='.git' \
-		--dereference \
-		--exclude='.gitignore' \
-		--exclude='demos' \
-		--exclude='.github' \
-		--transform s/qm/qm-${VERSION}/ \
-		-f /tmp/v${VERSION}.tar.gz ../qm
-	mv /tmp/v${VERSION}.tar.gz ./rpm
+                --exclude='.git' \
+                --dereference \
+                --exclude='.gitignore' \
+                --exclude='demos' \
+                --exclude='.github' \
+                --transform s/qm/qm-${VERSION}/ \
+                -f $${HOME}/rpmbuild/SOURCES/v${VERSION}.tar.gz ../qm
```

Comment:
.../qm is still failing.
Please work with static dir paths.
It is failing on in rpm build and in github actions check